shell/components/form/SSHKnownHosts/__tests__/KnownHostsEditDialog.test.ts

```diff
@@ -20,8 +20,9 @@ describe('component: KnownHostsEditDialog', () => {
     wrapper = mount(KnownHostsEditDialog, {
       attachTo: document.body,
       props:    {
-        mode:  _EDIT,
-        value: 'line1\nline2\n',
+        mode:                         _EDIT,
+        value:                        'line1\nline2\n',
+        disableFocusTrapForUnitTests: true
```

Comment:
I think that it makes more sense to attempt to mock the `focus-trap` module instead of introducing code that caters to unit tests within our components. For example, I was able to replace this instance of `disableFocusTrapForUnitTests` with the following jest mock:

```js
jest.mock('focus-trap', () => {
  return {
    createFocusTrap: jest.fn().mockImplementation(() => {
      return {
        activate:   jest.fn(),
        deactivate: jest.fn(),
      };
    }),
  };
});
```

Reply:
Man, this is great! Totally forgot about mocks! Way better! 💪 Thank you! 🙇